README.md

```diff
@@ -43,9 +43,13 @@
+Welcome to the repository for the Hierarchical Engine for Large-scale Infrastructure Co-Simulation (HELICS). HELICS provides an open-source, general-purpose, modular, highly-scalable co-simulation framework that runs cross-platform (Linux, Windows, and Mac OS X). It is not a modeling tool by itself, but rather an integration tool that enables multiple existing simulation tools (and/or multiple instances of the same tool), known as "federates", to exchange data during runtime and stay synchronized in time such that together they act as one large simulation, or "federation". This enables bringing together established (or new/emerging) off-the-shelf tools from multiple domains to form a complex software-simulation without having to change the individual tools (known as "black-box" modeling). All that is required is for someone to write a thin interface layer for each tool that interfaces with existing simulation time control and data value updating, such as through an existing scripting interface. Moreover, the HELICS community has a growing ecosystem of existing interfaces for popular tools, such that many users can simply mix and match existing tools with their own data and run complex co-simulations with minimal coding.
 
-**Brief History:** HELICS began as the core software development of the Grid Modernization Laboratory Consortium ([GMLC](https://gridmod.labworks.org/)) project on integrated Transmission-Distribution-Communication simulation (TDC, GMLC project 1.4.15) supported by the U.S. Department of Energy's Offices of Electricity Delivery and Energy Reliability ([OE](https://www.energy.gov/oe/office-electricity-delivery-and-energy-reliability)) and Energy Efficiency and Renewable Energy ([EERE](https://www.energy.gov/eere/office-energy-efficiency-renewable-energy)). As such, its first use cases center around modern electric power systems, though it can  and has been used for co-simulation in other domains. HELICS's layered, high-performance, co-simulation framework builds on the collective experience of multiple national labs.
+Today the core uses for HELICS are in the energy domain, where there is extensive and growing support for a wide-range of electric power system, communications and control-schemes, natural gas, transportation, built environment, and related domain tools. However, it is possible to use HELICS for co-simulation in any domain. Previous and existing HELICS efforts have stretched across a wide range of scales in time—e.g. from transient dynamics (e.g. power system frequency response or electromechanical transient simulation) to long-term planning studies—and space—from individual appliances to nationwide simulations.
```

Comment:
What is `built environment`? Build environments?

I found the end of the line about "in time" and "in space" a bit confusing to read, probably from the arrangement of -'s and nested e.g.'s (forgot about it being an example for "in time" when I reached the "-and space-" part of it).

Reply:
Ah, learned a new term and found out what a "built environment" is.

Reply:
Built environment is generalization for buildilngs/ Clarified